### ISSUE TITLE
Memory Leak?

### ISSUE THREAD

**Issue by BlankRain** [oakes/Nightcode#146]
I build the jar by myself.
```
git clone "https://github.com/oakes/Nightcode.git" 
cd Nightcode
lein deps
lein uberjar
```
Now,I got a runnable jar file.
On windows  10 os,Jvm 8, I double click it .
and then I quit it.
The TaskManager show that the JVM still running. 
That 's all.

(Sorry, I try to upload images,I failed)

**Comment by oakes** [comment]
Did you run any of the build commands? I've heard that on Windows, they may continue running after stopping them.

**Comment by oakes** [comment]
See this issue: https://github.com/oakes/Nightcode/issues/59

**Comment by BlankRain** [comment]
I do nothing...
I just open it and close it ...
There is no other opetations. No porject. No build command.No repl runs.

**Comment by oakes** [comment]
OK then perhaps the lower-left REPL is still running after quitting. I'll have to look into this.

**Comment by oakes** [comment]
I just released 1.0.1 which hopefully resolves this problem. Let me know if it's still happening.

**Comment by BlankRain** [comment]
I'm  sorry about long time no response for this issue.

I checked it and it  works. 

You resolved this problem.

Thanks.
